README.md

```diff
@@ -1 +1,112 @@
-# My_Portfolio
\ No newline at end of file
+# My_Portfolio
+
+# Hello-World
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="images/my_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My Portfolio README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 Hello World <a name="about-project"></a>
```

Comment:
- [optional] you can remove hello world and name it as your portfolio or with your name

Reply:
Thank you for the feedback @tufoinnkuo10 
I made the corrections and will be submitting the project again for a 👍🏼